openedx/core/djangoapps/user_authn/views/login_form.py

```diff
@@ -36,9 +38,102 @@
 from third_party_auth import pipeline
 from third_party_auth.decorators import xframe_allow_whitelisted
 
+from util.password_policy_validators import (
+    DEFAULT_MAX_PASSWORD_LENGTH,
+)
+
 log = logging.getLogger(__name__)
 
 
+def _apply_third_party_auth_overrides(request, form_desc):
```

Comment:
Are these two methods just a straight move, or do I need to review the code at all?

Reply:
These are a straight move. I did a little bit of fixing/cleanup, but I would probably like to do more later (the imports are kind of a shitshow right now).